app/win/BUILD.gn

```diff
@@ -53,8 +48,12 @@ copy("dist_resources") {
 
   outputs = [ "$brave_dist_dir/{{source_file_part}}" ]
 
-  public_deps = [
-    "//chrome/chrome_elf",
-    "//third_party/angle:copy_compiler_dll",
-  ]
+  public_deps = [ "//chrome/chrome_elf" ]
+
+  # Angle does not define copy_compiler_dll for arm64. The reason for this is
+  # that Windows versions that support arm64 already ship the associated file
+  # d3dcompiler_47.dll. So Chromium and we do not need to build it on arm64.
+  if (target_cpu != "arm64") {
+    public_deps += [ "//third_party/angle:copy_compiler_dll" ]
```

Comment:
this should be a data dep, but I don't understand why we need this. It's encapsulated in `libEGL` and `libGLESv2` so we should have deps on those

Reply:
Thank you @bridiver; To test your hypothesis, do you know whether it's enough to remove the `"//third_party/angle:copy_compiler_dll"` dep and see if CI still passes? Or are there additional things to test / investigate?

Reply:
@mherrmann I assume it should fail to build if there's a problem, but otherwise check for the output of `copy_compiler_dll` to make sure the dll is where it's supposed to be?

Reply:
@bridiver the build passed and the output of `copy_compiler_dll`, which is `d3dcompiler_47.dll`, still ends up in Brave's installation directory.

Resolving this conversation.